src/analysis/retail/evoker/preservation/modules/talents/Spiritbloom.tsx

```diff
@@ -29,6 +35,48 @@ class DreamBreath extends Analyzer {
     );
   }
 
+  get guideSubsection(): JSX.Element {
+    const explanation = (
+      <p>
+        <b>
+          <SpellLink id={TALENTS_EVOKER.SPIRITBLOOM_TALENT.id} />
+        </b>{' '}
+        is one of your empowered abilities and a very strong AoE triage heal that applies a powerful
+        HoT with the Abberus (T30) tier set. You should try to use this ability at maximum
+        Empowerment level whenever it is not on cooldown.{' '}
+        <SpellLink id={TALENTS_EVOKER.SPIRITBLOOM_TALENT} /> is a strong candidate to consume{' '}
+        <SpellLink spell={TALENTS_EVOKER.ECHO_TALENT} /> with, especially with the T30 tierset, but
+        may vary based on playstyle.
+      </p>
+    );
+
+    const data = (
```

Comment:
I think a qualitative performance bar similar to the dream breath guideSubsection would be good here, given the importance of hitting full empowered SBs bc of the tier set. Won't hold the pr up for it though